api.go

```diff
@@ -1290,8 +1290,8 @@ func (d *Context) SafeCall(fn, args *[0]byte, nargs, nrets int) int {
 }
 
 // See: http://duktape.org/api.html#duk_safe_to_lstring
-func (d *Context) SafeToLstring(index int, outLen int) string {
-	if s := C.duk_safe_to_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) SafeToLstring(index int, outLen *int) string {
```

Comment:
Last one, in general, totally cool with changing the api, but we need to version it for breaking changes. So if we don't need to pass an int pointer let's change it back to an int please.